Gemfile.lock

```diff
@@ -1,33 +1,41 @@
 PATH
   remote: .
   specs:
-    signature (0.1.8)
+    signature (0.1.9)
```

Comment:
Weird that this wasn't set to a version that matched the `lib/signature/version.rb` one already.

On that subject, I see that you appear to have pushed a pre-release version as `1.0.0...`. I think I'd probably go with `0.2.0` for this, since it doesn't seem like it actually has any compatibility-breaking changes? (Not that it matters too much, since we're the only consumers of this version of the gem.)